show/main.py

```diff
@@ -2028,6 +2028,29 @@ def ztp(status, verbose):
        cmd = cmd + " --verbose"
     run_command(cmd, display_cmd=verbose)
 
+#
+# 'static anycast gateway' command ("show static-anycast-gateway")
+#
+@cli.command('static-anycast-gateway')
+@clicommon.pass_db
+def sag(db):
+    """Show static anycast gateway information"""
+    sag_entry = db.cfgdb.get_entry('SAG', 'GLOBAL')
```

Comment:
what if SAG is not configured?

Reply:
This is changed in the updated PR https://github.com/sonic-net/sonic-utilities/pull/2881